src/main/java/rx/Observable.java

```diff
@@ -10699,14 +10702,14 @@ public final Subscription subscribe(Subscriber<? super T> subscriber) {
      *  <dd>The operator doesn't interfere with backpressure which is determined by the source {@code Observable}'s backpressure
      *  behavior.</dd>
      *  <dt><b>Scheduler:</b></dt>
-     *  <dd>{@code timeInterval} operates by default on the {@code immediate} {@link Scheduler}.</dd>
+     *  <dd>{@code timeInterval} operates by default on the {@code computation} {@link Scheduler}.</dd>
```

Comment:
Actually, this operator does not "operate" on the scheduler, it uses it only as source of time